newIDE/app/src/GameDashboard/GameDashboardCard.js

```diff
@@ -513,9 +518,34 @@ const GameDashboardCard = ({
                 // Extract word translation to ensure it is not wrongly translated in the sentence.
                 const translatedConfirmText = i18n._(t`delete`);
 
+                const hasPlayerMessage = countOfSessionsLastWeek
+                  ? t`${countOfSessionsLastWeek} views on the last 7 days`
+                  : t`No players`;
+                const hasBeenPublished = isPublishedOnGdGames
+                  ? t`Is published on gd.games`
+                  : t`Not published`;
+
+                const message = t`You're deleting a game that has:${'\n\n'}
```

Comment:
I would change the text because here you can have a case: 

`You're deleting a game that has: ... - is published`

I would suggest:

`You're deleting a game which: - has x views... - is published...`